iotdb-cli/src/main/java/org/apache/iotdb/cli/client/AbstractClient.java

```diff
@@ -132,7 +132,7 @@
   protected static int[] maxValueLengthForShow = new int[]{75, 45, 8, 8};
   protected static String formatTime = "%" + maxTimeLength + "s|";
   protected static String formatValue = "%" + maxValueLength + "s|";
-  protected static int DIVIDING_LINE_LEN = 40;
+  protected static int dividinglinelen = 40;
```

Comment:
all lower case?

Reply:
I guess it is because this is not a final field.

Reply:
I means that this variable should satisfy the name of Lower Camel Case....

Reply:
Maybe dividingLineLen is better?